### ISSUE TITLE
undefined method `primary_keys' for #<ActiveRecord::ConnectionAdapters::FirebirdAdapter:0x00007f9431a1b4c8> Did you mean?  primary_key

### ISSUE THREAD

**Issue by fwidtmann** [FabioMR/firebird_adapter#4]
hi,
i try to get my rails 4 app to rails 5, so i need to use your fb adapter.

but i get stuck with this error

```
undefined method `primary_keys' for #<ActiveRecord::ConnectionAdapters::FirebirdAdapter:0x00007f9431a1b4c8>
Did you mean?  primary_key
```
everytime i want to access the db like. Article.first

any ideas?

**Comment by FabioMR** [comment]
This error occur just when you change the table default name (self.table_name = 'NEW_NAME').

**Comment by FabioMR** [comment]
@fwidtmann I've just released a new version (1.0.5) that fix this bug!
